deploy.sh

```diff
@@ -29,10 +24,24 @@ main() {
 		elif [[ $1 = "-n" || $1 = "--no-hash" ]]; then
 			append_hash=false
 			shift
+		elif [[ $1 = "-c" || $1 = "--config-file" ]]; then
```

Comment:
that's good enough for me. i think it's better to give some kind of error than silently ignore a missing parameter.